src/controller/gtfs-pathways-controller.ts

```diff
@@ -65,41 +99,56 @@ class GtfsPathwaysController implements IController {
         }
     }
 
+    /**
+     * Function to create record in the database and upload the gtfs-pathway files
+     * @param request 
+     * @param response 
+     * @param next 
+     * @returns 
+     */
     createGtfsPathway = async (request: Request, response: express.Response, next: NextFunction) => {
         try {
-            let pathways = PathwayVersions.from(request.body);
+            console.log('Received upload request');
+            const meta = JSON.parse(request.body['meta']);
+            const userId = request.body.user_id;
+            // Validate the meta data
+            const gtfsdto = GtfsPathwaysUploadMeta.from(meta);
+            const result = await validate(gtfsdto);
+            console.log('result', result);
+        
+            if(result.length != 0){
+                console.log('Metadata validation failed');
+                console.log(result);
+                // Need to send these as response
```

Comment:
Please check the other pull request for flex. The meta errors are sent back

Reply:
fixed